.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
well, to be honest I really prefer to have requires inside files with `require_relative` as other YaST modules have it, this way it is easy to run something like `rspec path_to_file` regardless what is your working dir.